### DISCUSSION TITLE
Is there a way to listen to the audio that is sent to deepgram?

### DISCUSSION THREAD

**Discussion by team-deepgram** [deepgram/community#401] [General help] [answered]
Additionally, is there a way to listen to the audio that is sent to deepgram? For debuggin purposes

> [Posted by punk3730](https://discord.com/channels/1108042150941294664/1168833009722609687/1168833009722609687)

**Comment by team-deepgram** [comment, marked as answer]
Hi @tq there is no such functionality. We do not store the audio in such a manner that it is then easily accessible



> [Posted by lukeocodes](https://discord.com/channels/1108042150941294664/1168833009722609687/1168833208721346580)

**Comment by segheysens** [comment]
Hi Deepgram team, has this changed at all? I was hoping to use one of your realtime streaming examples, but need to keep the recording of the audio to, and make it accessible to my users.
